app/policies/form_policy.rb

```diff
@@ -58,7 +58,19 @@ def can_add_page_routing_conditions?
     form_has_two_or_more_pages && form_has_qualifying_pages
   end
 
-  alias_method :can_make_form_live?, :can_change_form_submission_email?
+  def can_make_form_live?
+    return can_change_form_submission_email? unless FeatureService.enabled?(:groups)
+
+    if form.group.active? && can_administer_group?
+      can_change_form_submission_email?
+    else
+      false
+    end
+  end
```

Comment:
I think this would be the guard clausey version:
```suggestion
  def can_make_form_live?
    return can_change_form_submission_email? unless FeatureService.enabled?(:groups)
    return can_change_form_submission_email? if form.group.active? && can_administer_group?
    
    false
  end
```

Reply:
I think that looks nicer...I'll do that 👍